### ISSUE TITLE
Tooltip on ComboBox is not read by screen reader

### ISSUE THREAD

**Issue by JingyuShao** [microsoft/fluentui#21446]
I created a code pen repro for this. Basically, when our site uses the tooltip on ComboBox components, the tooltip won't be read by screen reader when tabbed into (while the tooltip is definitely visibly)

[ComboBox autocomplete (codepen.io)](https://codepen.io/JingyuS/pen/XWgJPQo?editors=1010)

Please help to see how to fix the issue
Discussed with Micah about the issue on Teams channel, and he found 
"The problem is that the aria-describedby is being added to the root element instead of the input
    
    const divProps = getNativeProps<React.HTMLAttributes<HTMLDivElement>>(this.props, divProperties, [
      'onChange',
      'value',
    ]);
"

**Comment by spmonahan** [comment]
Hi @JingyuShao thanks for filing this issue and providing a repro example!

I [forked your example](https://codepen.io/seanms/pen/zYPvOQv?editors=0010) and changed

`aria-describedby={tooltipId}`

to

`ariaDescribedBy={tooltipId}`

(should be on L57 of the JS for the Codepen). I tested in Edge on Windows 11 with Narrator and the tooltip was read as expected.

The latter version is the one [defined in the ComboBox API](https://developer.microsoft.com/en-us/fluentui#/controls/web/combobox#implementation) that will correctly [apply the ID to the input](https://github.com/microsoft/fluentui/blob/master/packages/react/src/components/ComboBox/ComboBox.tsx#L600).
